### ISSUE TITLE
Docker 2.5.0.0 Crashes on startup

### ISSUE THREAD

**Issue by prime23-gareth** [docker/for-win#9428]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 7A65DEF7-D853-4A4A-AA22-E626EB7D505C/20201106214254

### Information

  - Windows Version: 10. Version 2004.
  - Docker Desktop Version: 2.5.0.0
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: NO

### Steps to reproduce the behavior

Literally just attempt to start Docker.

**Comment by adi-mohan** [comment]
Try to update Windows to 20H2 version. Updating to this version worked for me.

**Comment by prime23-gareth** [comment]
The problem was that the LxssManager had become disabled (I think due to my AntiVirus system disabling it) and this was stopping the WSL system running correctly. Access was denied to restart it. To resolve this I uninstalled the Windows Subsystem for Linux, rebooted and re-installed. (I also uninstalled/re-installed Docker and the Windows Container feature for good measure).
After that it all started working again.
